sources/packages/api/src/services/application/application.service.ts

```diff
@@ -497,20 +497,7 @@ export class ApplicationService extends RecordDataModelService<Application> {
   async getApplicationById(applicationId: number): Promise<Application> {
     return this.repo
       .createQueryBuilder("application")
-      .select([
-        "application.data",
-        "programYear.programYear",
-        "programYear.startDate",
-        "programYear.endDate",
-        "offering",
-        "offering.educationProgram",
-        "educationProgram.credentialType",
-        "educationProgram.completionYears",
-        "location.data",
-        "institution",
-        "institutionType",
-        "student",
-      ])
+      .select(["student.sin", "student.birthDate", "user.id", "user.lastName"])
```

Comment:
It seems that we are missing a lot of fields that were previously returned. Was it expected?

Reply:
looks like other fields were use for the validation checks was removed in @henryoforeh-aot  previous PR.  anyway please double-check @henryoforeh-aot

Reply:
@andrewsignori-aot @ann-aot is correct. The deleted fields were used in my previous PR that had some validation codes implemented in the now deleted validation-error-utils.ts file